perfeed/tools/pr_summarizer.py

```diff
@@ -44,5 +44,5 @@ async def run(self, repo: str, pr_number: int) -> PRSummary:
 
 if __name__ == "__main__":
     summarizer = PRSummarizer(GithubProvider("Perfeed"), llm=OllamaClient("llama3.2"))
-    pr_summary = summarizer.run("perfeed", 5)
+    pr_summary = asyncio.run(summarizer.run("perfeed", 5))
```

Comment:
Only one `asyncio.run()` is needed for the entire script. 
Would this be the one we want to put `asyncio.run()`?

Reply:
Yeah, what we want here is to run an async func in a non-async func since we cannot use `await` in a non-async func. This is exactly the use case for `asyncio.run`